src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
You should link to the compiled CSS file, not the SCSS source file. Use `href="styles/main.css"` instead of `href="styles/main.scss"`. SCSS files need to be compiled into CSS before they can be used in a browser.